### PR TITLE
Tomhaile/ch9229/remove ipfs

### PR DESCRIPTION
**Pull request by bthaile**
[Clubhouse Story](https://app.clubhouse.io/augur/story/9229)

Check to make sure the Dockerfile changes work with build process.
Check that documentation works.

Make sure we no longer are using deploy.js

## Submitter checklist
- [ ] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass

### CODE REVIEW

File: support/local-run.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd /etc/nginx/
+
+if [ ! -f cache ]; then
```
Comment (pgebheim) [inline support/local-run.sh]: These three lines can be replaced with

`mkdir -p cache`

File: support/nginx-default.conf [hunk 2]
```diff
@@ -6,26 +6,10 @@ server {
   gzip_vary on;
   gzip_min_length 10240;
   gzip_proxied expired no-cache no-store private auth;
-  gzip_types text/plain text/css text/xml text/html text/javascript application/x-javascript application/xml;
+  gzip_types text/plain text/css text/xml text/javascript application/x-javascript application/xml;
```
Comment (pgebheim) [inline support/nginx-default.conf]: gotta keep text/html in the gzip section here.

Comment (bthaile) [reply support/nginx-default.conf]: I was getting a warning so I removed it. It's included as a gzip type by default. 
http://nginx.org/en/docs/http/ngx_http_gzip_module.html

Comment (pgebheim) [reply support/nginx-default.conf]: oh! ok cool beans then.

**Pull request merged**